packages/devtools/web/index.html

```diff
@@ -105,6 +102,22 @@
 
             Plotly.extendTraces(domName, data, traces);
         }
+
+        function extendTraces4(domName, x0, x1, x2, x3, y0, y1, y2, y3, orgTraces) {
```

Comment:
can you file a bug to track down why these workarounds are needed. It would be nice to be able to put all 4 methods in Dart code. On the surface I expect that is possible without much extra effort.